src/include/stir/scatter/ScatterEstimation.h

```diff
@@ -171,6 +171,8 @@ class ScatterEstimation : public ParsingObject
 
   inline void set_num_iterations(int);
 
+  inline void set_downsample_scanner(bool downsample_scanner, int downsampled_detectors_per_ring = 0);
```

Comment:
once we do this, it would make most sense to add the variable for the "dets per ring" as well. 

normally default variables like this should be -1, such that we know it's invalid. The code indeed checks for <= 0, so it's fine, but I'd prefer -1...